src/main/scala/co/teapot/tempest/graph/DirectedGraph.scala

```diff
@@ -44,6 +48,11 @@ trait DirectedGraph {
     * */
   def inNeighbors(id: Int): IndexedSeq[Int]
 
+  def neighbors(id: Int, direction: EdgeDir): Int = direction match {
+    case EdgeDirOut => outDegree(id)
```

Comment:
Good catch!  I copy-pasted `degree` too quickly.
I fixed this and added tests for it.